### ISSUE TITLE
Problem uploading iOS ipa file to App Store (Invalid bundle structure)

### ISSUE THREAD

**Issue by branomalovec** [JimmyPun610/BarcodeScanner.Mobile#152]
We're having an issue when using Transporter app to deploy ipa file with BarcodeScanner.Mobile.XamarinForms package. It is "Invalid bundle structure..." error with GoogleMaps, see screenshot. Our project was uploadable before, only change was to add BarcodeScanner.Mobile.XamarinForms, no code changes. I have tried all suggestions i found on the internet but nothing have helped. Any help with this?

I was really impressed how good was scanning on Android comparing to ZXing and would like to use a package for iOS too.

![Bundle_issue](https://user-images.githubusercontent.com/104762344/200003752-54b98e62-1c51-481e-b6dd-a72498f6cb57.PNG)

**Comment by JimmyPun610** [comment]
interesting...this library should not have anything related to GoogleMap.
Did you find the cause?

**Comment by branomalovec** [comment]
No, we didn't figure out how to get over this issue. Had to use other scanning solution.
